PCF-ADR-0004-codeflare-test-strategy.md

```diff
@@ -0,0 +1,392 @@
+# Define testing strategy for Project CodeFlare
+
+|                |                          |
+| -------------- | ------------------------ |
+| Date           | 06/14/2023               |
+| Scope          |                          |
+| Status         | Proposed                 |
+| Authors        | [Karel Suta](@sutaakar), [Antonin Stefanutti](@astefanutti)  |
+| Supersedes     | N/A                      |
+| Superseded by: | N/A                      |
+| Issues         |                          |
+| Other docs:    | N/A                      |
```

Comment:
Could you link to https://github.com/project-codeflare/adr/blob/main/PCF-ADR-0003-codeflare-release-process.md here?

Reply:
Done